2024Robot_Team3484/src/main/cpp/subsystems/LauncherSubsystem.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or Launcherare it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "subsystems/LauncherSubsystem.h"
+
+#include <numbers>
+
+using namespace LauncherConstants;
+using namespace rev;
+
+LauncherSubsystem::LauncherSubsystem(
+        int MOTOR_LEFT_CAN_ID, 
+        int MOTOR_RIGHT_CAN_ID,
+        double P_Launcher,
+        double I_Launcher,
+        double D_Launcher,
+        double FF_Launcher,
+        double RPM_Window_Launcher 
+    ):
+    _left_motor{MOTOR_LEFT_CAN_ID, rev::CANSparkMax::MotorType::kBrushless},
+    _right_motor{MOTOR_RIGHT_CAN_ID, rev::CANSparkMax::MotorType::kBrushless}
+    {
+    Launcher_Encoder_Left = new SparkRelativeEncoder(_left_motor.GetEncoder(rev::SparkRelativeEncoder::Type::kQuadrature, 4096 ));
+    Launcher_Encoder_Right = new SparkRelativeEncoder(_right_motor.GetEncoder(rev::SparkRelativeEncoder::Type::kQuadrature, 4096 ));
+
+    Launcher_m_Left_pidController = _left_motor.GetPIDController();
+    Launcher_m_Right_pidController = _right_motor.GetPIDController();
```

Comment:
I suggest updating the PID controller assignments to match the encoder assignments above.